Output not cropped and squashed with full QSV path in nightlies

### Problem description:

When running with QSV decoding and QSV encoding, the crop doesn't seem to be applied and videos are squashed vertically

It works in 1.3.3, although that uses the "encode only path"

### HandBrake version (e.g., 1.3.0):

`HandBrake 20201117184955-a2716d5db-master`

### Operating system and version (e.g., Ubuntu 19.10, macOS 10.15 Catalina, Windows 10 1909):

Windows 10 20H2

### Error message text and/or screenshot:

Input:
![vlcsnap-2020-11-19-10h14m05s076](https://user-images.githubusercontent.com/13986933/99652702-100d7600-2a50-11eb-8a51-d68c8a8d687a.png)

Output:
![vlcsnap-2020-11-19-10h13m22s009](https://user-images.githubusercontent.com/13986933/99652728-1996de00-2a50-11eb-94ed-64f7a350739e.png)


### HandBrake Activity Log:

~~~
[09:35:09] Compile-time hardening features are enabled
Cannot load nvEncodeAPI64.dll
Cannot load nvEncodeAPI64.dll
Cannot load nvEncodeAPI64.dll
Cannot load nvEncodeAPI64.dll
[09:35:10] hb_init: starting libhb thread
[09:35:10] thread 1 started ("libhb")
HandBrake 20201117184955-a2716d5db-master (2020111801) - MinGW x86_64 - https://handbrake.fr
8 CPUs detected
Opening D:\Films\Brave\Brave (2012).mkv...
[09:35:10] CPU: Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
[09:35:10]  - Intel microarchitecture Kaby Lake
[09:35:10]  - logical processor count: 8
[09:35:10] Intel Quick Sync Video support: yes, gpu list: 0
[09:35:10]  - Intel Media SDK hardware: API 1.30 (minimum: 1.3)
[09:35:10]  - H.264 encoder: yes
[09:35:10]     - preferred implementation: hardware (any) via D3D11
[09:35:10]     - capabilities (hardware):  breftype icq+la+i+downs vsinfo opt1 opt2+mbbrc+extbrc+trellis+ib_adapt+nmpslice
[09:35:10]  - H.265 encoder: yes (8bit: yes, 10bit: yes)
[09:35:10]     - preferred implementation: hardware (any) via D3D11
[09:35:10]     - capabilities (hardware):  bpyramid icq vsinfo opt1
[09:35:10] hb_scan: path=D:\Films\Brave\Brave (2012).mkv, title_index=1
udfread ERROR: ECMA 167 Volume Recognition failed
src/libbluray/disc/disc.c:323: failed opening UDF image D:\Films\Brave\Brave (2012).mkv
src/libbluray/disc/disc.c:424: error opening file BDMV\index.bdmv
src/libbluray/disc/disc.c:424: error opening file BDMV\BACKUP\index.bdmv
src/libbluray/bluray.c:2585: nav_get_title_list(D:\Films\Brave\Brave (2012).mkv\) failed
[09:35:10] bd: not a bd - trying as a stream/file instead
libdvdnav: Using dvdnav version 6.0.1
libdvdread: Encrypted DVD support unavailable.
libdvdread:DVDOpenFileUDF:UDFFindFile /VIDEO_TS/VIDEO_TS.IFO failed
libdvdread:DVDOpenFileUDF:UDFFindFile /VIDEO_TS/VIDEO_TS.BUP failed
libdvdread: Can't open file VIDEO_TS.IFO.
libdvdnav: vm: failed to read VIDEO_TS.IFO
[09:35:10] dvd: not a dvd - trying as a stream/file instead
Input #0, matroska,webm, from 'D:\Films\Brave\Brave (2012).mkv':
  Metadata:
    encoder         : libebml v1.4.0 + libmatroska v1.6.2
    creation_time   : 2020-10-14T10:37:45.000000Z
  Duration: 01:33:37.32, start: 0.000000, bitrate: 28065 kb/s
    Chapter #0:0: start 0.000000, end 262.845917
    Metadata:
      title           : Chapter 01
    Chapter #0:1: start 262.845917, end 557.431875
    Metadata:
      title           : Chapter 02
    Chapter #0:2: start 557.431875, end 773.355917
    Metadata:
      title           : Chapter 03
    Chapter #0:3: start 773.355917, end 840.256083
    Metadata:
      title           : Chapter 04
    Chapter #0:4: start 840.256083, end 970.469500
    Metadata:
      title           : Chapter 05
    Chapter #0:5: start 970.469500, end 1056.680625
    Metadata:
      title           : Chapter 06
    Chapter #0:6: start 1056.680625, end 1375.791083
    Metadata:
      title           : Chapter 07
    Chapter #0:7: start 1375.791083, end 1642.891250
    Metadata:
      title           : Chapter 08
    Chapter #0:8: start 1642.891250, end 1731.229500
    Metadata:
      title           : Chapter 09
    Chapter #0:9: start 1731.229500, end 1854.853000
    Metadata:
      title           : Chapter 10
    Chapter #0:10: start 1854.853000, end 2040.371667
    Metadata:
      title           : Chapter 11
    Chapter #0:11: start 2040.371667, end 2163.494667
    Metadata:
      title           : Chapter 12
    Chapter #0:12: start 2163.494667, end 2346.927917
    Metadata:
      title           : Chapter 13
    Chapter #0:13: start 2346.927917, end 2515.095917
    Metadata:
      title           : Chapter 14
    Chapter #0:14: start 2515.095917, end 2607.980375
    Metadata:
      title           : Chapter 15
    Chapter #0:15: start 2607.980375, end 2825.739583
    Metadata:
      title           : Chapter 16
    Chapter #0:16: start 2825.739583, end 2902.357792
    Metadata:
      title           : Chapter 17
    Chapter #0:17: start 2902.357792, end 3064.061000
    Metadata:
      title           : Chapter 18
    Chapter #0:18: start 3064.061000, end 3153.859042
    Metadata:
      title           : Chapter 19
    Chapter #0:19: start 3153.859042, end 3241.154583
    Metadata:
      title           : Chapter 20
    Chapter #0:20: start 3241.154583, end 3285.699083
    Metadata:
      title           : Chapter 21
    Chapter #0:21: start 3285.699083, end 3450.864083
    Metadata:
      title           : Chapter 22
    Chapter #0:22: start 3450.864083, end 3540.161625
    Metadata:
      title           : Chapter 23
    Chapter #0:23: start 3540.161625, end 3768.181083
    Metadata:
      title           : Chapter 24
    Chapter #0:24: start 3768.181083, end 3807.386917
    Metadata:
      title           : Chapter 25
    Chapter #0:25: start 3807.386917, end 4197.735208
    Metadata:
      title           : Chapter 26
    Chapter #0:26: start 4197.735208, end 4237.274708
    Metadata:
      title           : Chapter 27
    Chapter #0:27: start 4237.274708, end 4355.726375
    Metadata:
      title           : Chapter 28
    Chapter #0:28: start 4355.726375, end 4419.498417
    Metadata:
      title           : Chapter 29
    Chapter #0:29: start 4419.498417, end 4535.322458
    Metadata:
      title           : Chapter 30
    Chapter #0:30: start 4535.322458, end 4619.072792
    Metadata:
      title           : Chapter 31
    Chapter #0:31: start 4619.072792, end 4657.110792
    Metadata:
      title           : Chapter 32
    Chapter #0:32: start 4657.110792, end 4788.200083
    Metadata:
      title           : Chapter 33
    Chapter #0:33: start 4788.200083, end 4996.783458
    Metadata:
      title           : Chapter 34
    Chapter #0:34: start 4996.783458, end 5078.656917
    Metadata:
      title           : Chapter 35
    Chapter #0:35: start 5078.656917, end 5572.191625
    Metadata:
      title           : Chapter 36
    Chapter #0:36: start 5572.191625, end 5617.344000
    Metadata:
      title           : Chapter 37
    Stream #0:0(eng): Video: h264 (High), yuv420p(progressive), 1920x1080 [SAR 1:1 DAR 16:9], 23.98 fps, 23.98 tbr, 1k tbn, 47.95 tbc (default)
    Metadata:
      BPS-eng         : 23778255
      DURATION-eng    : 01:33:37.320000000
      NUMBER_OF_FRAMES-eng: 134681
      NUMBER_OF_BYTES-eng: 16696258791
      SOURCE_ID-eng   : 001011
      _STATISTICS_WRITING_APP-eng: mkvmerge v50.0.0 ('Awakenings') 64-bit
      _STATISTICS_WRITING_DATE_UTC-eng: 2020-10-14 10:37:45
      _STATISTICS_TAGS-eng: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES SOURCE_ID
    Stream #0:1(eng): Audio: truehd, 48000 Hz, 7.1, s32 (24 bit) (default)
    Metadata:
      title           : Surround 7.1
      BPS-eng         : 4210702
      DURATION-eng    : 01:33:37.321000000
      NUMBER_OF_FRAMES-eng: 6740785
      NUMBER_OF_BYTES-eng: 2956608236
      SOURCE_ID-eng   : 001100
      _STATISTICS_WRITING_APP-eng: mkvmerge v50.0.0 ('Awakenings') 64-bit
      _STATISTICS_WRITING_DATE_UTC-eng: 2020-10-14 10:37:45
      _STATISTICS_TAGS-eng: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES SOURCE_ID
    Stream #0:2(eng): Subtitle: hdmv_pgs_subtitle, 1920x1080
    Metadata:
      BPS-eng         : 23629
      DURATION-eng    : 01:33:06.644000000
      NUMBER_OF_FRAMES-eng: 1774
      NUMBER_OF_BYTES-eng: 16501410
      SOURCE_ID-eng   : 001201
      _STATISTICS_WRITING_APP-eng: mkvmerge v50.0.0 ('Awakenings') 64-bit
      _STATISTICS_WRITING_DATE_UTC-eng: 2020-10-14 10:37:45
      _STATISTICS_TAGS-eng: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES SOURCE_ID
[09:35:10] scan: decoding previews for title 1
[09:35:10] scan: audio 0x1: truehd, rate=48000Hz, bitrate=1 English (TrueHD) (7.1 ch)

Scanning title 1 of 1, preview 2, 20.00 %
Scanning title 1 of 1, preview 3, 30.00 %
Scanning title 1 of 1, preview 5, 50.00 %
Scanning title 1 of 1, preview 6, 60.00 %
Scanning title 1 of 1, preview 8, 80.00 %
Scanning title 1 of 1, preview 9, 90.00 %[09:35:11] scan: 10 previews, 1920x1080, 23.976 fps, autocrop = 138/140/0/0, aspect 16:9, PAR 1:1
[09:35:11] scan: supported video decoders: avcodec qsv
[09:35:11] libhb: scan thread found 1 valid title(s)
+ Using preset: CLI Default
+ title 1:
  + stream: D:\Films\Brave\Brave (2012).mkv
  + duration: 01:33:37
  + size: 1920x1080, pixel aspect: 1/1, display aspect: 1.78, 23.976 fps
  + autocrop: 138/140/0/0
  + chapters:
    + 1: duration 00:04:23
    + 2: duration 00:04:55
    + 3: duration 00:03:36
    + 4: duration 00:01:07
    + 5: duration 00:02:10
    + 6: duration 00:01:26
    + 7: duration 00:05:19
    + 8: duration 00:04:27
    + 9: duration 00:01:28
    + 10: duration 00:02:04
    + 11: duration 00:03:06
    + 12: duration 00:02:03
    + 13: duration 00:03:03
    + 14: duration 00:02:48
    + 15: duration 00:01:33
    + 16: duration 00:03:38
    + 17: duration 00:01:17
    + 18: duration 00:02:42
    + 19: duration 00:01:30
    + 20: duration 00:01:27
    + 21: duration 00:00:45
    + 22: duration 00:02:45
    + 23: duration 00:01:29
    + 24: duration 00:03:48
    + 25: duration 00:00:39
    + 26: duration 00:06:30
    + 27: duration 00:00:40
    + 28: duration 00:01:58
    + 29: duration 00:01:04
    + 30: duration 00:01:56
    + 31: duration 00:01:24
    + 32: duration 00:00:38
    + 33: duration 00:02:11
    + 34: duration 00:03:29
    + 35: duration 00:01:22
    + 36: duration 00:08:14
    + 37: duration 00:00:45
  + audio tracks:
    + 1, English (TrueHD) (7.1 ch) (iso639-2: eng)
  + subtitle tracks:
    + 1, English [PGS]
[09:35:11] Starting work at: Thu Nov 19 09:35:11 2020

[09:35:11] 1 job(s) to process
[09:35:11] json job:
{
    "Audio": {
        "AudioList": [
            {
                "Bitrate": 640,
                "Encoder": 2048,
                "Track": 0
            }
        ],
        "CopyMask": [
            "copy:aac",
            "copy:ac3",
            "copy:eac3",
            "copy:dtshd",
            "copy:dts",
            "copy:mp3",
            "copy:truehd",
            "copy:flac"
        ],
        "FallbackEncoder": "av_aac"
    },
    "Destination": {
        "AlignAVStart": false,
        "ChapterList": [
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 4,
                    "Seconds": 23,
                    "Ticks": 23656132
                },
                "Name": "Chapter 1"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 4,
                    "Seconds": 55,
                    "Ticks": 26512736
                },
                "Name": "Chapter 2"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 3,
                    "Seconds": 36,
                    "Ticks": 19433164
                },
                "Name": "Chapter 3"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 1,
                    "Seconds": 7,
                    "Ticks": 6021015
                },
                "Name": "Chapter 4"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 2,
                    "Seconds": 10,
                    "Ticks": 11719208
                },
                "Name": "Chapter 5"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 1,
                    "Seconds": 26,
                    "Ticks": 7759001
                },
                "Name": "Chapter 6"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 5,
                    "Seconds": 19,
                    "Ticks": 28719941
                },
                "Name": "Chapter 7"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 4,
                    "Seconds": 27,
                    "Ticks": 24039015
                },
                "Name": "Chapter 8"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 1,
                    "Seconds": 28,
                    "Ticks": 7950443
                },
                "Name": "Chapter 9"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 2,
                    "Seconds": 4,
                    "Ticks": 11126115
                },
                "Name": "Chapter 10"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 3,
                    "Seconds": 6,
                    "Ticks": 16696679
                },
                "Name": "Chapter 11"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 2,
                    "Seconds": 3,
                    "Ticks": 11081070
                },
                "Name": "Chapter 12"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 3,
                    "Seconds": 3,
                    "Ticks": 16508993
                },
                "Name": "Chapter 13"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 2,
                    "Seconds": 48,
                    "Ticks": 15135120
                },
                "Name": "Chapter 14"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 1,
                    "Seconds": 33,
                    "Ticks": 8359601
                },
                "Name": "Chapter 15"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 3,
                    "Seconds": 38,
                    "Ticks": 19598329
                },
                "Name": "Chapter 16"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 1,
                    "Seconds": 17,
                    "Ticks": 6895639
                },
                "Name": "Chapter 17"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 2,
                    "Seconds": 42,
                    "Ticks": 14553289
                },
                "Name": "Chapter 18"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 1,
                    "Seconds": 30,
                    "Ticks": 8081823
                },
                "Name": "Chapter 19"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 1,
                    "Seconds": 27,
                    "Ticks": 7856599
                },
                "Name": "Chapter 20"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 0,
                    "Seconds": 45,
                    "Ticks": 4009005
                },
                "Name": "Chapter 21"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 2,
                    "Seconds": 45,
                    "Ticks": 14864850
                },
                "Name": "Chapter 22"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 1,
                    "Seconds": 29,
                    "Ticks": 8036779
                },
                "Name": "Chapter 23"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 3,
                    "Seconds": 48,
                    "Ticks": 20521751
                },
                "Name": "Chapter 24"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 0,
                    "Seconds": 39,
                    "Ticks": 3528525
                },
                "Name": "Chapter 25"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 6,
                    "Seconds": 30,
                    "Ticks": 35131346
                },
                "Name": "Chapter 26"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 0,
                    "Seconds": 40,
                    "Ticks": 3558555
                },
                "Name": "Chapter 27"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 1,
                    "Seconds": 58,
                    "Ticks": 10660650
                },
                "Name": "Chapter 28"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 1,
                    "Seconds": 4,
                    "Ticks": 5739484
                },
                "Name": "Chapter 29"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 1,
                    "Seconds": 56,
                    "Ticks": 10424164
                },
                "Name": "Chapter 30"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 1,
                    "Seconds": 24,
                    "Ticks": 7537530
                },
                "Name": "Chapter 31"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 0,
                    "Seconds": 38,
                    "Ticks": 3423420
                },
                "Name": "Chapter 32"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 2,
                    "Seconds": 11,
                    "Ticks": 11798036
                },
                "Name": "Chapter 33"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 3,
                    "Seconds": 29,
                    "Ticks": 18772504
                },
                "Name": "Chapter 34"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 1,
                    "Seconds": 22,
                    "Ticks": 7368611
                },
                "Name": "Chapter 35"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 8,
                    "Seconds": 14,
                    "Ticks": 44418124
                },
                "Name": "Chapter 36"
            },
            {
                "Duration": {
                    "Hours": 0,
                    "Minutes": 0,
                    "Seconds": 45,
                    "Ticks": 4061644
                },
                "Name": "Chapter 37"
            }
        ],
        "ChapterMarkers": true,
        "File": "Brave (2012).mkv",
        "InlineParameterSets": false,
        "Mp4Options": {
            "IpodAtom": false,
            "Mp4Optimize": false
        },
        "Mux": "mkv"
    },
    "Filters": {
        "FilterList": [
            {
                "ID": 6,
                "Settings": {
                    "mode": 0
                }
            },
            {
                "ID": 12,
                "Settings": {
                    "crop-bottom": 140,
                    "crop-left": 0,
                    "crop-right": 0,
                    "crop-top": 138,
                    "height": 802,
                    "width": 1920
                }
            }
        ]
    },
    "Metadata": {},
    "PAR": {
        "Den": 1,
        "Num": 1
    },
    "SequenceID": 0,
    "Source": {
        "Angle": 0,
        "Path": "D:\\Films\\Brave\\Brave (2012).mkv",
        "Range": {
            "End": 37,
            "Start": 1,
            "Type": "chapter"
        },
        "Title": 1
    },
    "Subtitle": {
        "Search": {
            "Burn": true,
            "Default": false,
            "Enable": false,
            "Forced": false
        },
        "SubtitleList": [
            {
                "Burn": false,
                "Default": false,
                "Forced": false,
                "Track": 0
            }
        ]
    },
    "Video": {
        "Bitrate": 8000,
        "ColorFormat": 0,
        "ColorMatrix": 1,
        "ColorPrimaries": 1,
        "ColorRange": 0,
        "ColorTransfer": 1,
        "Encoder": "qsv_h264",
        "Level": "4.0",
        "Profile": "high",
        "QSV": {
            "AsyncDepth": 4,
            "Decode": true
        },
        "Turbo": false,
        "TwoPass": false
    }
}
[09:35:11] Starting Task: Encoding Pass
[09:35:11] Skipping vfr filter
[09:35:11] qsv: integrated qsv adapter with index 0 has been selected
[09:35:11] work: mixdown not specified, track 1 setting mixdown 5.1 Channels
[09:35:11] job configuration:
[09:35:11]  * source
[09:35:11]    + D:\Films\Brave\Brave (2012).mkv
[09:35:11]    + title 1, chapter(s) 1 to 37
[09:35:11]    + container: matroska,webm
[09:35:11]    + data rate: 28065 kbps
[09:35:11]  * destination
[09:35:11]    + Brave (2012).mkv
[09:35:11]    + container: Matroska (libavformat)
[09:35:11]      + chapter markers
[09:35:11]  * video track
[09:35:11]    + decoder: h264_qsv
[09:35:11]    + filters
[09:35:11]      + Crop and Scale (width=1920:height=802:crop-top=138:crop-bottom=140:crop-left=0:crop-right=0)
[09:35:11]        + source: 1920 * 1080, crop (138/140/0/0): 1920 * 802, scale: 1920 * 802
[09:35:11]    + Output geometry
[09:35:11]      + storage dimensions: 1920 x 802
[09:35:11]      + pixel aspect ratio: 1 : 1
[09:35:11]      + display dimensions: 1920 x 802
[09:35:11]    + encoder: H.264 (Intel Media SDK)
[09:35:11]      + profile: high
[09:35:11]      + level:   4.0
[09:35:11]      + bitrate: 8000 kbps, pass: 0
[09:35:11]      + color profile: 1-1-1
[09:35:11]  * subtitle track 1, English [PGS] (track 0, id 0x2, Picture) -> Passthrough
[09:35:11]  * audio track 1
[09:35:11]    + decoder: English (TrueHD) (7.1 ch) (track 1, id 0x1)
[09:35:11]      + samplerate: 48000 Hz
[09:35:11]    + mixdown: 5.1 Channels
[09:35:11]    + encoder: AC3 (libavcodec)
[09:35:11]      + bitrate: 640 kbps, samplerate: 48000 Hz
[09:35:11] sync: expecting 134681 video frames
[09:35:12] encqsvInit: using full QSV path
[09:35:12] encqsvInit: H.264/AVC High profile @ level 4.0
[09:35:12] encqsvInit: TargetUsage 4 AsyncDepth 4
[09:35:12] encqsvInit: GopRefDist 3 GopPicSize 24 NumRefFrame 2
[09:35:12] encqsvInit: BFramesMax 2 BRefType off
[09:35:12] encqsvInit: RateControlMethod LA TargetKbps 8000 LookAheadDepth 40
[09:35:12] sync: first pts audio 0x1 is 0
[09:35:12] sync: first pts subtitle 0x2 is 90090
[09:35:12] sync: first pts video is 0
[09:35:12] sync: "Chapter 1" (1) at frame 1 time 0
[AVHWFramesContext @ 0000000006ae5440] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000006ae3fc0] Error initializing a child frames context
[AVHWFramesContext @ 0000000006ae3fc0] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 0000000006ae4600] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000006ae3c40] Error initializing a child frames context
[AVHWFramesContext @ 0000000006ae3c40] Error creating an internal frame pool
[09:35:12] qsv_enc_init: using 'hardware (1) via D3D11' implementation, API: 1.30
[AVHWFramesContext @ 0000000006ae5300] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000006ae3b00] Error initializing a child frames context
[AVHWFramesContext @ 0000000006ae3b00] Error creating an internal frame pool
[AVHWFramesContext @ 0000000006ae3e40] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000006ae5b40] Error initializing a child frames context
[AVHWFramesContext @ 0000000006ae5b40] Error creating an internal frame pool
[09:36:06] sync: "Chapter 2" (2) at frame 6303 time 23656132
[AVHWFramesContext @ 000000000dbb8780] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000dbb9800] Error initializing a child frames context
[AVHWFramesContext @ 000000000dbb9800] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 3 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000dbb6ec0] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000dbb9680] Error initializing a child frames context
[AVHWFramesContext @ 000000000dbb9680] Error creating an internal frame pool
[09:37:02] sync: "Chapter 3" (3) at frame 13366 time 50168868
[AVHWFramesContext @ 000000000d09ccc0] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000d09c940] Error initializing a child frames context
[AVHWFramesContext @ 000000000d09c940] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000d09c940] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000d09ccc0] Error initializing a child frames context
[AVHWFramesContext @ 000000000d09ccc0] Error creating an internal frame pool
[09:37:43] sync: "Chapter 4" (4) at frame 18543 time 69602032
[AVHWFramesContext @ 0000000012250000] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000012250340] Error initializing a child frames context
[AVHWFramesContext @ 0000000012250340] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 00000000122508c0] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000012250400] Error initializing a child frames context
[AVHWFramesContext @ 0000000012250400] Error creating an internal frame pool
[09:37:56] sync: "Chapter 5" (5) at frame 20149 time 75630555
[AVHWFramesContext @ 000000000df5ff00] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000df5f380] Error initializing a child frames context
[AVHWFramesContext @ 000000000df5f380] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000df5ffc0] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000df5ff00] Error initializing a child frames context
[AVHWFramesContext @ 000000000df5ff00] Error creating an internal frame pool
[09:38:21] sync: "Chapter 6" (6) at frame 23269 time 87342255
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[09:38:37] sync: "Chapter 7" (7) at frame 25336 time 95101256
[AVHWFramesContext @ 0000000011b28680] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000011b293c0] Error initializing a child frames context
[AVHWFramesContext @ 0000000011b293c0] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 0000000011b293c0] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000011b28800] Error initializing a child frames context
[AVHWFramesContext @ 0000000011b28800] Error creating an internal frame pool
[09:39:38] sync: "Chapter 8" (8) at frame 32989 time 123828705
[AVHWFramesContext @ 0000000011b29240] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000011b28ac0] Error initializing a child frames context
[AVHWFramesContext @ 0000000011b28ac0] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 0000000011b29300] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000011b28ec0] Error initializing a child frames context
[AVHWFramesContext @ 0000000011b28ec0] Error creating an internal frame pool
[09:40:30] sync: "Chapter 9" (9) at frame 39392 time 147863966
[AVHWFramesContext @ 000000000e0c3700] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000e0c3640] Error initializing a child frames context
[AVHWFramesContext @ 000000000e0c3640] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000e0c3700] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000e0c3640] Error initializing a child frames context
[AVHWFramesContext @ 000000000e0c3640] Error creating an internal frame pool
[09:40:47] sync: "Chapter 10" (10) at frame 41509 time 155810655
[AVHWFramesContext @ 0000000006ae2840] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000006ae2f40] Error initializing a child frames context
[AVHWFramesContext @ 0000000006ae2f40] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 0000000006ae1a80] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000006ae3080] Error initializing a child frames context
[AVHWFramesContext @ 0000000006ae3080] Error creating an internal frame pool
[09:41:11] sync: "Chapter 11" (11) at frame 44473 time 166936770
[AVHWFramesContext @ 000000000e24b1c0] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000e24ab00] Error initializing a child frames context
[AVHWFramesContext @ 000000000e24ab00] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000e24ac40] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000e24a400] Error initializing a child frames context
[AVHWFramesContext @ 000000000e24a400] Error creating an internal frame pool
[09:41:46] sync: "Chapter 12" (12) at frame 48921 time 183633450
[AVHWFramesContext @ 000000000e2c69c0] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000e2c6900] Error initializing a child frames context
[AVHWFramesContext @ 000000000e2c6900] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000e2c69c0] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000e2c6900] Error initializing a child frames context
[AVHWFramesContext @ 000000000e2c6900] Error creating an internal frame pool
[09:42:10] sync: "Chapter 13" (13) at frame 51873 time 194714520
[AVHWFramesContext @ 000000000e2c6f40] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000e2c6e80] Error initializing a child frames context
[AVHWFramesContext @ 000000000e2c6e80] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000e2c6f40] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000e2c6e80] Error initializing a child frames context
[AVHWFramesContext @ 000000000e2c6e80] Error creating an internal frame pool
[09:42:44] sync: "Chapter 14" (14) at frame 56271 time 211223512
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[09:43:17] sync: "Chapter 15" (15) at frame 60303 time 226358632
[AVHWFramesContext @ 000000000cf919c0] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000cf92200] Error initializing a child frames context
[AVHWFramesContext @ 000000000cf92200] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000cf919c0] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000cf91f40] Error initializing a child frames context
[AVHWFramesContext @ 000000000cf91f40] Error creating an internal frame pool
[09:43:34] sync: "Chapter 16" (16) at frame 62532 time 234725741
[AVHWFramesContext @ 000000000cf92200] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000cf91b00] Error initializing a child frames context
[AVHWFramesContext @ 000000000cf91b00] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000cf91d00] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000cf92200] Error initializing a child frames context
[AVHWFramesContext @ 000000000cf92200] Error creating an internal frame pool
[09:44:16] sync: "Chapter 17" (17) at frame 67751 time 254316562
[AVHWFramesContext @ 0000000006bdd180] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000006bdd7c0] Error initializing a child frames context
[AVHWFramesContext @ 0000000006bdd7c0] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 0000000006bdd040] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000006bdd180] Error initializing a child frames context
[AVHWFramesContext @ 0000000006bdd180] Error creating an internal frame pool
[09:44:31] sync: "Chapter 18" (18) at frame 69588 time 261212201
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[09:45:01] sync: "Chapter 19" (19) at frame 73465 time 275765490
[AVHWFramesContext @ 0000000049ef2400] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000049ef2b00] Error initializing a child frames context
[AVHWFramesContext @ 0000000049ef2b00] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 0000000049ef2580] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000049ef2200] Error initializing a child frames context
[AVHWFramesContext @ 0000000049ef2200] Error creating an internal frame pool
[09:45:18] sync: "Chapter 20" (20) at frame 75619 time 283851067
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[09:45:35] sync: "Chapter 21" (21) at frame 77711 time 291703912
[AVHWFramesContext @ 000000004c862f00] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000004c862dc0] Error initializing a child frames context
[AVHWFramesContext @ 000000004c862dc0] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000004c862840] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000004c862f00] Error initializing a child frames context
[AVHWFramesContext @ 000000004c862f00] Error creating an internal frame pool
[09:45:45] sync: "Chapter 22" (22) at frame 78781 time 295720425
[AVHWFramesContext @ 000000000cd80340] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000cd802c0] Error initializing a child frames context
[AVHWFramesContext @ 000000000cd802c0] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000cd80840] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000cd802c0] Error initializing a child frames context
[AVHWFramesContext @ 000000000cd802c0] Error creating an internal frame pool
[09:46:16] sync: "Chapter 23" (23) at frame 82741 time 310585275
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[09:46:33] sync: "Chapter 24" (24) at frame 84880 time 318614546
[AVHWFramesContext @ 000000000cf92080] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000cf91fc0] Error initializing a child frames context
[AVHWFramesContext @ 000000000cf91fc0] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000cf91a40] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000cf91bc0] Error initializing a child frames context
[AVHWFramesContext @ 000000000cf91bc0] Error creating an internal frame pool
[09:47:23] sync: "Chapter 25" (25) at frame 90348 time 339140051
[AVHWFramesContext @ 000000000cf91b00] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000cf92140] Error initializing a child frames context
[AVHWFramesContext @ 000000000cf92140] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000cf91b00] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000cf92140] Error initializing a child frames context
[AVHWFramesContext @ 000000000cf92140] Error creating an internal frame pool
[09:47:32] sync: "Chapter 26" (26) at frame 91287 time 342664822
[AVHWFramesContext @ 000000000cf91b00] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000cf91440] Error initializing a child frames context
[AVHWFramesContext @ 000000000cf91440] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[09:48:47] sync: "Chapter 27" (27) at frame 100648 time 377803676
[AVHWFramesContext @ 000000004a78d2c0] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000004a78d640] Error initializing a child frames context
[AVHWFramesContext @ 000000004a78d640] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000004a78d580] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000004a78ddc0] Error initializing a child frames context
[AVHWFramesContext @ 000000004a78ddc0] Error creating an internal frame pool
[09:48:55] sync: "Chapter 28" (28) at frame 101594 time 381354723
[AVHWFramesContext @ 000000004a78da40] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000004a78dd00] Error initializing a child frames context
[AVHWFramesContext @ 000000004a78dd00] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000004a78d2c0] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000004a78de80] Error initializing a child frames context
[AVHWFramesContext @ 000000004a78de80] Error creating an internal frame pool
[09:49:17] sync: "Chapter 29" (29) at frame 104436 time 392022881
[AVHWFramesContext @ 0000000012079b00] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000012079dc0] Error initializing a child frames context
[AVHWFramesContext @ 0000000012079dc0] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 0000000012079b00] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000001207a800] Error initializing a child frames context
[AVHWFramesContext @ 000000001207a800] Error creating an internal frame pool
[09:49:30] sync: "Chapter 30" (30) at frame 105964 time 397758611
[AVHWFramesContext @ 0000000033caa4c0] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000033caadc0] Error initializing a child frames context
[AVHWFramesContext @ 0000000033caadc0] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 0000000033cab080] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000033caafc0] Error initializing a child frames context
[AVHWFramesContext @ 0000000033caafc0] Error creating an internal frame pool
[09:49:52] sync: "Chapter 31" (31) at frame 108742 time 408186528
[AVHWFramesContext @ 0000000033caa580] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000033cab280] Error initializing a child frames context
[AVHWFramesContext @ 0000000033cab280] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 0000000033caad00] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000033cab280] Error initializing a child frames context
[AVHWFramesContext @ 0000000033cab280] Error creating an internal frame pool
[09:50:08] sync: "Chapter 32" (32) at frame 110748 time 415716551
[AVHWFramesContext @ 0000000033caa580] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000033caae80] Error initializing a child frames context
[AVHWFramesContext @ 0000000033caae80] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 0000000033caac40] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000033caad00] Error initializing a child frames context
[AVHWFramesContext @ 0000000033caad00] Error creating an internal frame pool
[09:50:15] sync: "Chapter 33" (33) at frame 111660 time 419139971
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[09:50:40] sync: "Chapter 34" (34) at frame 114805 time 430945515
[AVHWFramesContext @ 0000000006e52b00] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000006e53b80] Error initializing a child frames context
[AVHWFramesContext @ 0000000006e53b80] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 0000000006e53840] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000006e53600] Error initializing a child frames context
[AVHWFramesContext @ 0000000006e53600] Error creating an internal frame pool
[09:51:20] sync: "Chapter 35" (35) at frame 119806 time 449718018
[AVHWFramesContext @ 000000000dd39900] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000dd390c0] Error initializing a child frames context
[AVHWFramesContext @ 000000000dd390c0] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000dd394c0] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000dd39380] Error initializing a child frames context
[AVHWFramesContext @ 000000000dd39380] Error creating an internal frame pool
[09:51:36] sync: "Chapter 36" (36) at frame 121767 time 457079122
[AVHWFramesContext @ 000000000dd39380] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000dd38b40] Error initializing a child frames context
[AVHWFramesContext @ 000000000dd38b40] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 000000000dd38b40] Unsupported pixel format: pal8
[AVHWFramesContext @ 000000000dd38e00] Error initializing a child frames context
[AVHWFramesContext @ 000000000dd38e00] Error creating an internal frame pool
[09:53:09] sync: "Chapter 37" (37) at frame 133600 time 501497246
[AVHWFramesContext @ 0000000006ae3840] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000006ae3780] Error initializing a child frames context
[AVHWFramesContext @ 0000000006ae3780] Error creating an internal frame pool
ERROR: Error initializing a frames context for an internal frame allocation request

ERROR: Last error repeated 1 times
ERROR: Error initializing a frames context for an internal frame allocation request

[AVHWFramesContext @ 0000000006ae3780] Unsupported pixel format: pal8
[AVHWFramesContext @ 0000000006ae2440] Error initializing a child frames context
[AVHWFramesContext @ 0000000006ae2440] Error creating an internal frame pool
[09:53:18] reader: done. 1 scr changes
[09:53:18] work: average encoding speed for job is 123.978256 fps
[09:53:18] truehd-decoder done: 6740785 frames, 0 decoder errors
[09:53:18] h264_qsv-decoder done: 134681 frames, 0 decoder errors
[09:53:18] sync: got 134681 frames, 134681 expected
[09:53:18] sync: framerate min 23.976 fps, max 23.976 fps, avg 23.976 fps
[09:53:18] mux: track 0, 134681 frames, 5616170370 bytes, 7998.30 kbps, fifo 512
[09:53:18] mux: track 1, 175541 frames, 449384960 bytes, 639.99 kbps, fifo 1024
[09:53:18] mux: track 2, 1774 frames, 16501410 bytes, 23.50 kbps, fifo 16
[09:53:18] Finished work at: Thu Nov 19 09:53:18 2020

[09:53:18] libhb: work result = 0

Encode done!
HandBrake has exited.
~~~

--- a/libhb/cropscale.c
+++ b/libhb/cropscale.c
@@ -107,6 +107,14 @@ static int crop_scale_init(hb_filter_object_t * filter, hb_filter_init_t * init)
             return result;
         }
 
+        if (top > 0 || bottom > 0 || left > 0 || right > 0)
+        {
+            hb_dict_set_int(avsettings, "cx", left);
+            hb_dict_set_int(avsettings, "cy", top);
+            hb_dict_set_int(avsettings, "cw", cropped_width);
+            hb_dict_set_int(avsettings, "ch", cropped_height);
+        }
+
         hb_dict_set_int(avsettings, "w", width);
         hb_dict_set_int(avsettings, "h", height);
         if (init->job->qsv.ctx->vpp_scale_mode)
@@ -119,7 +127,7 @@ static int crop_scale_init(hb_filter_object_t * filter, hb_filter_init_t * init)
         }
         hb_log("qsv: scaling filter mode %s", init->job->qsv.ctx->vpp_scale_mode ? init->job->qsv.ctx->vpp_scale_mode : "default");
         hb_log("qsv: scaling filter interpolation method %s", init->job->qsv.ctx->vpp_interpolation_method ? init->job->qsv.ctx->vpp_interpolation_method : "default");
-        hb_dict_set(avfilter, "scale_qsv", avsettings);
+        hb_dict_set(avfilter, "vpp_qsv", avsettings);
 
         AVHWFramesContext *frames_ctx;
         AVQSVFramesContext *frames_hwctx;

